src/components/account/AccountSetting.jsx

```diff
@@ -0,0 +1,99 @@
+import { useState } from "react";
+
+const AccountSetting = () => {
+  const [phoneNumber, setPhoneNumber] = useState("");
+
+  const formatPhoneNumber = (input) => {
```

Comment:
nice work here! Might be good to add the utils dir as other components will likely need it.

Reply:
Hey Bruce, I agree with Christian here since my registration form also has the phone number input. If you could pull this function into a utils folder in a future PR, I'd appreacite it so that I can call it on my form as well.